web/src/components/zonelist.js

```diff
@@ -93,6 +95,7 @@ const ZoneList = ({
 
   const enterZone = (zone) => {
     dispatchApplication('mapViewport', getCenteredZoneViewport(zone));
+    trackEvent('ZoneInRanking Clicked', {zone: zone.countryCode});
```

Comment:
Do you still believe having separate events for this is the best way forward? I'm a bit unsure what we came to a conclusion about 🤔 

Suggestion for alternative naming if we use the same event:  
`'Zone Viewed', {zone: zone.countryCode, source: 'ranking_panel'}`
`'Zone Viewed', {zone: zone.countryCode, source: 'map'}`

(As I think the goal is to view a zone, not to click a link - I think 😅)

Reply:
Well, I think for consistency tracking the actions that a user can take and not the resulting experience. I.e. if we add an event for the wind panel, we would track "WindButton Clicked" and not the resulting experience (that the wind layer becomes enabled)

This means that all events would follow "Item + userAction". When that is said, I do like the idea of your events for this case though, I am not sure how important (or even possible) it is to maintain a 100 % consistent events

Reply:
Ah, now I read [the Plausible documentation](https://plausible.io/docs/custom-event-goals) and realised that `events !== goals` (we manually set those up). Also seems like I'm a bit stuck on how Amplitude is doing it/recommending it :)
Then I think your solution is fine, and I guess we can just track this specific link and based on that figure out how many people end on `/zone/DE` via this event or other means.